modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/raft/MetaStorageWriteHandler.java

```diff
@@ -88,13 +88,9 @@ void handleWriteCommand(CommandClosure<WriteCommand> clo) {
                 safeTime = cmdWithTime.safeTime();
 
                 handleWriteWithTime(clo, cmdWithTime, safeTime);
-
-                // Every MetaStorageWriteCommand holds safe time that we should set as the cluster time.
-                clusterTime.updateSafeTime(safeTime);
             } else if (command instanceof SyncTimeCommand) {
-                clusterTime.updateSafeTime(((SyncTimeCommand) command).safeTime());
-
-                clo.result(null);
+                // TODO: IGNITE-19199 WatchProcessor must be notified of the new safe time.
+                throw new UnsupportedOperationException("https://issues.apache.org/jira/browse/IGNITE-19199");
```

Comment:
So, `SyncTimeCommand` is not used anywhere right now, correct?

Reply:
Yes

Reply:
ok